src/dataDisplay/Number/index.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+
+const DEFAULT_SHOW_DECIMALS = true;
+const DEFAULT_SHOW_THOUSAND_SEPARATORS = false;
+
+type FormatNumberOptions = {
+  roundingType?: 'up' | 'down';
+  showDecimals?: boolean;
+  decimalPlaces?: number;
+  showThousandSeparators?: boolean;
+  currency?: string;
+  as?: 'p' | 'span';
+};
+
+export const formatNumber = (
```

Comment:
Is it possible to add a doc for this function? I assume it's necessary to use if you want to include text into the DOM without an HTML element

Reply:
I'll write one now.